src/components/molecules/Card/Card.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react'
+import styled from 'styled-components'
+
+// A T O M S
+import { Icon, IconName, IconSize } from '../../atoms/Icon/Icon'
+
+export interface CardProps {
+  className?: string
+  footer?: React.ReactNode
```

Comment:
Passing a component as a props is hard to maintain and to read in a complex jsx structure. I would rather use the composition pattern with another predefine component (ex: `CardFooter`):

```html
          <Card
            title="Card with Icon"
            icon="remove"
            iconSize='lg'
            linkUrl="https://youtube.com"
          >
            <CardFooter>
              <p>Just a footer</p>
            <CardFooter>
          </Card>
```

What's your opinion on this?